charts/vault/Makefile

```diff
@@ -0,0 +1,32 @@
+SHELL := /bin/bash # Use bash syntax
```

Comment:
What command requires it?

Reply:
The while in the wait-for-vault command. I will remove it based on your next suggestion.